ntbs-service/Migrations/20200304084430_UpdateExistingTreatmentRegimenAndDropOtherColumns.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class UpdateExistingTreatmentRegimenAndDropOtherColumns : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"  
+                UPDATE ClinicalDetails
+                SET TreatmentRegimen = 'MdrTreatment'
+                WHERE IsMDRTreatment = 1");           
+            
+            migrationBuilder.Sql(@"  
+                UPDATE ClinicalDetails
+                SET TreatmentRegimen = 'StandardTherapy'
+                WHERE IsShortCourseTreatment = 1");
+            
+            migrationBuilder.DropColumn(
+                name: "IsMDRTreatment",
+                table: "ClinicalDetails");
+
+            migrationBuilder.DropColumn(
+                name: "IsShortCourseTreatment",
+                table: "ClinicalDetails");
```

Comment:
These will fail since the migration above is still present. You'll want to either combine the two migrations or at least remove dropping from the previous migration